spec/lib/email_processor_spec.rb

```diff
@@ -166,4 +187,25 @@
       expect(described_class.new(email).retrieve_forwarder_email).to eq(expected_result)
     end
   end
+
+  describe '#retrieve_course_by_url' do
+    it 'should return the first slug from a message' do
+      body = <<~EXAMPLE
+        Example email test\r\n\r\nhttps://example.org/courses/example/slug
```

Comment:
We should test also that the right slug is found for a url like `https://dashboard.wikiedu.org/courses/School_of_the_Art_Institute_of_Chicago/New_Histories_of_Chicago_Media_(Spring_2019)/articles/edited?showArticle=51050930`